backend/src/test/java/backend/mulkkam/cup/domain/vo/CupAmountTest.java

```diff
@@ -0,0 +1,40 @@
+package backend.mulkkam.cup.domain.vo;
+
+import static backend.mulkkam.common.exception.BadRequestErrorCode.INVALID_CUP_AMOUNT;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import backend.mulkkam.common.exception.CommonException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class CupAmountTest {
+
+    @Nested
+    @DisplayName("생성자 검증 시에")
```

Comment:
```suggestion
    @DisplayName("생성자 검증 시에")
    @Nested
```

중요도가 낮은 것이 젤 위로~~ up up up